### PR TITLE
 Add day 1 task content (kids-882)

### PR DESCRIPTION
**Pull request by Daniela510**
## Description

- task content
- banner already works also, it takes the title of the task as the banner title unless provided otherwise. And in this case its the same.

### CODE REVIEW

**Review by TammiLion — APPROVED** [review]
Added a comment but otherwise looks ok.

File: lib/features/children/generosity_challenge/utils/generosity_challenge_content_helper.dart [hunk 1]
```diff
@@ -6,10 +6,10 @@ import 'package:givt_app/features/children/generosity_challenge/models/task.dart
 class GenerosityChallengeContentHelper {
   static final List<Task> _tasks = [
     Task.card(
-      image: 'assets/images/generosity_challenge_placeholder.svg',
-      title: 'In a small village',
+      image: 'assets/images/generosity_challenge_day_1.svg',
+      title: 'Save the letter',
       description:
-          "Lived a craftsman Geppetto. One day he decided to make a wooden toy. He said to himself, 'I will make a little boy and call him Pinocchio.'",
+          "Today's assignment is for each family member to answer the question on the Mayor's letter. Once you've done that, stick it on your fridge with the magnet where you will see it everyday!\n\nDone? Hit the Complete Button",
```
[the remark this answers is not in the document]
Comment (MaikelStuivenberg) [reply lib/features/children/generosity_challenge/utils/generosity_challenge_content_helper.dart:12]: @TammiLion according to the design it's on :) But we can ask Andy

[the remark this answers is not in the document]
Comment (MaikelStuivenberg) [reply lib/features/children/generosity_challenge/utils/generosity_challenge_content_helper.dart:12]: I asked andy. It's **in** indeed

[the remark this answers is not in the document]
Comment (TammiLion) [reply lib/features/children/generosity_challenge/utils/generosity_challenge_content_helper.dart:12]: thanks for checking!

**Pull request merged**